### PR TITLE
BLS Integration 2nd attempt

### PR DESCRIPTION
**Pull request by kilic**

### CODE REVIEW

File: contracts/DepositManager.sol [hunk 1]
```diff
@@ -75,53 +73,21 @@ contract DepositManager {
...
 
     /**
      * @notice Adds a deposit for an address to the deposit queue
-     * @param _destination Address for which we are depositing
      * @param _amount Number of tokens that user wants to deposit
      * @param _tokenType Type of token user is depositing
      */
     function depositFor(
-        address _destination,
+        uint256 accountID,
```
Comment (vaibhavchellani) [inline contracts/DepositManager.sol:84]: How do I get an account ID to send here?

**Comment by vaibhavchellani** [comment]
LGTM.

File: contracts/DepositManager.sol (hunk 1 above)
Comment (kilic) [reply contracts/DepositManager.sol:84]: Depositor has to be registered to BLSAccountRegistery in advance

**Comment by kilic** [comment]
> LGTM. Need to fix tests.

PR is not ready to merge

**Pull request merged**